scripts/userstyles.yml

```diff
@@ -450,6 +452,13 @@ userstyles:
     readme:
       app-link: "https://hoppscotch.io/"
     current-maintainers: [*justtobbi]
+  learnxinyminutes:
```

Comment:
```suggestion
  learn-x-in-y-minutes:
```

Reply:
Make sure to change the name of the directory holding the userstyle and the metadata.